README.md

```diff
@@ -86,10 +86,6 @@ The API documentation is organized by components:
 * [VASTParser](docs/api/vast-parser.md)
 * [VASTTracker](docs/api/vast-tracker.md)
 
-**:warning: IMPORTANT :warning:** : the release of the `3.0` version of the library introduced many breaking changes in the API.
-
-Read the [3.0 migration guide](docs/api/3.0-migration.md) to update your project or follow the [2.0 migration guide](docs/api/2.0-migration.md) or [1.x API documentation](docs/api/1.x) if you're still using the old version.
-
```

Comment:
I think we should keep an indirect link to these files inside README, with the release entry point for example ?
I'm thinking of something like :

### Changelog 
Changelog & migration guides can be found in the [release notes](https://github.com/dailymotion/vast-client-js/releases).